user/tests/wiring/no_fixture/application.cpp

```diff
@@ -2,10 +2,16 @@
 #include "unit-test/unit-test.h"
 
 // make clean all TEST=wiring/no_fixture PLATFORM=electron -s COMPILE_LTO=n program-dfu DEBUG_BUILD=y
+// make clean all TEST=wiring/no_fixture PLATFORM=electron -s COMPILE_LTO=n program-dfu DEBUG_BUILD=y USE_THREADING=y
```

Comment:
Nice addition here with `USE_THREADING=y`, this is something added to release process just now.  Some of the tests already have threading turned on by default i.e., `wiring/threading` but this is something we'd like to start regularly testing with all existing tests originally written with a single thread in mind.